src/orion/algo/dehb/brackets.py

```diff
@@ -0,0 +1,42 @@
+"""Specialization of SHBracketManager to support duplicated trials"""
+
+import logging
+
+from orion.algo.dehb.logger import remove_loguru
+
+remove_loguru()
+
+
+try:
+    from dehb.utils import SHBracketManager as SHBracketManagerImpl
+
+    IMPORT_ERROR = None
+except ImportError as exc:
+    IMPORT_ERROR = exc
+
+
+logger = logging.getLogger(__name__)
+
+
+class SHBracketManager(SHBracketManagerImpl):
```

Comment:
This will cause an error if the try-import doesn't succeed, since `SHBracketManagerImpl` won't be defined. Same with the other try-import in the other file.

One solution is to just declare a dummy class for SHBracketManagerImpl inside the `except` block.

Reply:
Added in https://github.com/Epistimio/orion/commit/ebfff61c98c062202355f2ce3c308661e9fc80a1